heartbeat/iSCSILogicalUnit.in

```diff
@@ -43,6 +43,9 @@ elif have_binary lio_node; then
 	OCF_RESKEY_implementation_default="lio"
 elif have_binary targetcli; then
 	OCF_RESKEY_implementation_default="lio-t"
+	# guess "block", validate may fix that to be iblock
+	# alternatively, we could make this a parameter as well
+	BACKSTORES_BLOCK="/backstores/block"
```

Comment:
It looks like this wont be necessary, as the if/else in validate is run before start and stop, which are the only places the variable is used.